src/content/docs/release-notes/docs-release-notes/docs-8-2-2024.mdx

```diff
@@ -0,0 +1,109 @@
+---
+subject: "Docs"
+releaseDate: '2024-08-02'
+version: 'version: July 26-August 2, 2024'
+---
+
+### New docs
+
+* [Approaches to .NET serverless AWS Lambda performance monitoring](/docs/apm/agents/net-agent/getting-started/net-agent-approaches-lambda/) aims to clarify and outline the different requirements and capabilities when setting up instrumentation for your .NET Lambda functions.
+* You can monitor your [VMWare Tanzu NGINX integration](/docs/infrastructure/host-integrations/host-integrations-list/cloudfoundry-integrations/vmware-tanzu-nginx-buildpack-integration) with either .NET Core or Framework.
+
+
+### Major changes
+
+* [IAST supports Ruby](/docs/iast/introduction) in APM version 9.12.0 or higher.
+* Added a new customer responsibility matrix (CRM) spreadsheet to our [FedRAMP](/docs/security/security-privacy/compliance/certificates-standards-regulations/fedramp/) certification doc.
+* Added detailed info to [synthetics job manager configuration](/docs/synthetics/synthetic-monitoring/private-locations/job-manager-configuration/#user-file-example) on how to mount a `PersistentVolume` for a job manager on Kubernetes.
+* When you monitor your mobile app, you can disable distributed tracing as described in [configure mobile monitoring settings](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings/#configurations).
+* In [intro to Pathpoint](/docs/new-relic-solutions/business-observability/intro-pathpoint/#flows), added new best practices info around creating flows, staging examples, and selecting relevant signals.
```

Comment:
This reads kind of weird in lowercase, it reads like it should be an uppercase Intro. If not, maybe rewording to something like "In the intro to Pathpoint doc..." or something like that